docker/Dockerfile

```diff
@@ -112,6 +112,12 @@ RUN cd /tmp/osie && \
     cd && \
     rm -r /tmp/osie
 
+# IPMICFG
+COPY lfs/ipmicfg /tmp/osie/
```

Comment:
It'd be good to have in the commit message the url and version of this file

Reply:
Added 🙂